Vagrantfile

```diff
@@ -1,10 +1,13 @@
+
 hostname = "atlantis-aquarium-vm"
 domain   = "local"
 
 Vagrant.configure("2") do |config|
+  config.ssh.private_key_path = "./vagrant-id_rsa"
```

Comment:
I'm curious - why set a specific key?  In my experience, the default works fine.